playbooks/ansible-test-integration-base/run.yaml

```diff
@@ -33,6 +33,11 @@
         _test_options: "{{ _test_options }} --inventory /home/zuul/inventory"
       when: ansible_test_command == 'network-integration'
 
+    - name: Enable --retry-on-error
+      set_fact:
+        _test_options: "{{ _test_options }} --retry-on-error"
+      when: ansible_test_retry_on_error | default(False)
+
```

Comment:
I don't think we want to do this, if ansible-test cannot successful pass first round, we are likely exposing errors some place.  This will depend on us checking limestone.

This alone, likely means we have to make job non-voting by default.

Reply:
Yes, yes and no :-)

Until the root cause of the ESXI kernel panic is resolved we will see failure time to time. If the ESXi panic, it will reboot, and so we may have situation were the retry actually succeed.

So yes, it's not elegant, but I would prefer to keep it until the situation is resolved.

Reply:
I still think this is a blocker, if we are getting panic, the we need to focus on dealing with them. By enabling this, we are just going to ignore IMO

Reply:
Lets drop this, just to see how bad things are.  As I mentioned, I want to iterate on this if an issue. Given job is non-voting, we can see how bad it is.